### DISCUSSION TITLE
Update guideLink

### DISCUSSION THREAD

**Discussion by Mister-Hope** [DreamOfIce/vuepress-plugin-i18n#9] [Ideas]
Can we update the `guideLink` option into something like `translationGuide`?

Besides an internal route path under base locale, we can support external links like GitHub wiki, repo readme

**Comment by DreamOfIce** [comment]
I've recently been considering refactoring this section to simplify the code and support external links better

**Reply by DreamOfIce** [reply]
> I've recently been considering refactoring this section to simplify the code and support external links better

Done with d54251bbb23c1b4610d17bf453751d8c1544a416.
Now external links will have the attribute `target="_blank" `

**Comment by DreamOfIce** [comment]
> Can we update the `guideLink` option into something like `translationGuide`?

Done with 1326472ad576cc0f9de86adfe8211243033303a1
